platform-server-kernel/src/main/java/uno/cod/platform/server/core/dto/challenge/ChallengeCreateDto.java

```diff
@@ -10,6 +10,9 @@
     private String name;
 
     @NotNull
+    private Long organizationId;
+
```

Comment:
`If it means that anybody can do the challenge (without invitation), I am sure companies will like that.`
i'd say so, companies always create their own challenges i'd say, it doesn't make much sense to reuse them, since everyone can create organizations, so it's pretty much the same ¯\_(ツ)_/¯